### PR TITLE
Update_description_cli_operations

### PR DESCRIPTION
**Pull request by FranBarrera**
Remove description container.

### CODE REVIEW

File: cli_reference/basic_cli_operations.adoc [hunk 1]
```diff
@@ -323,7 +323,7 @@ work for other object types.
 |`exec`
 |`oc exec -p _<pod_ID>_ \`
 `-c _<container_ID>_ -- _<command>_`
-|Execute a command in a already-running container. It will default to the first container if none is specified.
```
Comment (adellape) [inline cli_reference/basic_cli_operations.adoc]: Is this no longer true?

File: cli_reference/basic_cli_operations.adoc [hunk 2]
```diff
@@ -323,7 +323,7 @@ work for other object types.
...
+|Execute a command in a already-running container.
```
Comment (adellape) [inline cli_reference/basic_cli_operations.adoc]: s/a/an/

**Comment by FranBarrera** [comment]
my fault, I wanted to remove only this part " It will default to the first container if none is specified".

**Comment by adellape** [comment]
> my fault, I wanted to remove only this part " It will default to the first container if none is specified".

I was trying to verify whether that was no longer true, but couldn't find anything to suggest things had changed. Can you clarify?

**Comment by FranBarrera** [comment]
Yes, for the "oc exec" command it is not necessary to add the option "-c container". As soon as I can check, if this still exists, I will notify.

**Comment by adellape** [comment]
It's definitely not required, but I don't know that it invalidates that it defaults to the first container when a container is not specified. `-c` is still a valid, optional flag, AFAICT. But maybe it's no longer worth mentioning.

**Comment by FranBarrera** [comment]
As it is not required, I think it makes no sense mentioning but if still exists then I can close this PR.

**Comment by adellape** [comment]
@FranBarrera On second thought, I'm fine w/ removing it. Still the s/a/an/ typo from above that can use addressing. Also the actual removal of the `-c-` option in the example is in this other PR, which has a minor space issue: https://github.com/openshift/openshift-docs/pull/1034#discussion-diff-41009572

**Comment by FranBarrera** [comment]
You're right, it's better in this PR https://github.com/openshift/openshift-docs/pull/1034#discussion-diff-41009572

**Pull request closed without merging**